.github/workflows/release_ci.yml

```diff
@@ -69,7 +69,7 @@ jobs:
         rm -rf ./src/OrchardCore.Cms.Web/App_Data_Tests
         dotnet publish -c Release -o ./.build/release --no-build --no-restore --framework net5.0
         docker build -f Dockerfile-CI -t orchardproject/orchardcore-cms-linux:dev .
-        docker push orchardproject/orchardcore-cms-linux
+        docker push orchardproject/orchardcore-cms-linux:dev
```

Comment:
Actually the issue might be that this will try always only update the `dev` tagged image while we would need to be able to eventually push an image for the `latest` tag.

Reply:
The issue is maybe directly in the Docker hub website settings for our images.

Reply:
I am getting the same issue here: https://github.com/StatCan/StatCan.OrchardCore/runs/2007163352?check_suite_focus=true#step:7:113